### DISCUSSION TITLE
Optional Test Data in Training Tab

### DISCUSSION THREAD

**Discussion by mariam-als** [birdnet-team/BirdNET-Analyzer#888] [Q&A]
Hello!

I am using BirdNET to develop classifiers for several baleen whale calls. My workflow consists of using the training tab to input clips I've collected, running batch analysis to generate prediction files, and evaluating performance in Raven and R. I was curious about the optional "Select Test Data" in the training tab. What can I input here and what does it generate? Does it influence my classifiers in any way?

Thank you,
Mariam Alsaid

**Comment by Josef-Haupt** [comment]
Hi! The wording as a bit misleading it's really validation data used to check the performance of your classifier.
Same as the training data directory, the test data directory should contain subfolders named accordingly after the species they contain.
If you don't provide additional test data, we use a small percentage of your training data for the validation. The validation is used during training to check if the classifier is able to learn what we want it to learn, basically if it's able to apply the learned patterns to unseen/unknown data. Having a sophisticated test set can really elevate the training process because it gives much more insight on how the model actually performs in the target scenario.
